src/util/Filter.js

```diff
@@ -446,35 +446,32 @@ function filterApply(card, filter, inCube) {
   if (filter.category == 'oracle' && card.details.oracle_text) {
     res = card.details.oracle_text.toLowerCase().indexOf(filter.arg) > -1;
   }
-  if (filter.category == 'color' && card.details.colors) {
+  if (filter.category == 'color' && colors !== undefined) {
```

Comment:
'colors' is defined above as the card's color_identiy, or override. We should be using card.details.colors for the color filter